### PR TITLE
Added html template file, server displays raw data, lots of updates to readme

### PR DESCRIPTION
**Pull request by jonathanzhang99**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-app=Flask(__name__)
+app = Flask(__name__)
+
 
 @app.route('/')
 def test():
-	return "TESTING TESTING TESTING"
-	
\ No newline at end of file
+    return "TESTING TESTING TESTING"
```
Comment (alexander-yu) [inline app.py]: It would be good to add a newline here at the end of the file.

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,5 @@
 *.proto
 *_pb2.py
 *.pyc
+staticTransit/*
```
Comment (alexander-yu) [inline .gitignore]: Could you change the name of the directory to be in all lowercase + underscores to follow Python style?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+ADI LEADERS - PLEASE LET US KNOW IF YOU ARE MAKING CHANGES TO THIS REPOSITORY - recent modifications have significantly disrupted workflow for the team. Thank you!
```
Comment (alexander-yu) [inline README.md]: The visibility change should be a one-time thing, so don't worry about this haha. You can go ahead and remove this.

**Comment by alexander-yu** [comment]
Looks like flake8 threw some style warnings concerning the imports. Could you go ahead and fix this? If not, I'll see if I can do it.

Also, in the meantime, would you mind squashing some of the formatting commits?

**Comment by jonathanzhang99** [comment]
flake8 can be overly pedantic at times especially with minor details. I think we can merge as is - some of the errors that it is throwing have very little effect on our code.

**Comment by alexander-yu** [comment]
The thing is, the concept of Travis (and CI in general) is that merging pull requests is blocked until a CI build passes, so we'll either have to pass the style check or add a flake8 exception in the YAML file. For now, I'll probably have it pass flake8 until we determine exactly which rules we want flake8 to skip.

**Pull request merged**